components/FilterControls.tsx

```diff
@@ -169,6 +169,26 @@ export function FilterControls({
           Resolved
         </button>
 
+        <button
+          onClick={() =>
+            setExtraFilters({
+              ...extraFilters,
+              readyToResolve: false,
+              resolvingSoon: false,
```

Comment:
Why set `resolvingSoon` to false, I don't think they are mutually exclusive filters? In fact, `resolvingSoon` does two things, it filters for unresolved questions and orders questions by resolve date (soonest first, hides questions where resolveBy is before today). So maybe when `resolvingSoon` is pressed, it should set `unresolved` = true?

This feels a bit messy. Another option is to change the "Resolved" button to a dropdown where you can select either Resolved or Unresolved. Or we could get rid of "Resolving soon", and add an "Order by" dropdown that lets where you can sort by resolution date either high-to-low, low-to-high, or creation date high-to-low or low-to-high

What do you think?

Reply:
(Remember to mirror any changes from here across to the other instance of the button below. I agree that this component could use a refactor!)

Reply:
I think an "Order by" dropdown _would_ be nice, but to get the tiny version of this feature out sooner I'm inclined to go with having `resolvingSoon` set `unresolved` for now. It already unsets `resolved`, so the new behavior will match that.